src/shared/services/commandUtils.test.js

```diff
@@ -115,4 +115,23 @@ describe('commandutils', () => {
       expect(obj.str + ': ' + utils.isCypherCommand(obj.str, ':')).toEqual(obj.str + ': ' + obj.expect)
     })
   })
+  test('extractPostConnectCommandsFromServerConfig should split and return an array of commands', () => {
+    // Given
+    const testStrs = [
+      { str: '', expect: undefined },
+      { str: ';;;;;;;;', expect: undefined },
+      { str: ':play cypher', expect: [':play cypher'] },
+      { str: '    :play cypher     ', expect: [':play cypher'] },
+      { str: ':play cypher;', expect: [':play cypher'] },
+      { str: ':play cypher ;', expect: [':play cypher'] },
+      { str: ';:play cypher;', expect: [':play cypher'] },
+      { str: ':play cypher; :param x: 1', expect: [':play cypher', ':param x: 1'] },
+      { str: 'RETURN 1; RETURN 3; :play start', expect: ['RETURN 1', 'RETURN 3', ':play start'] }
+    ]
```

Comment:
Splitting on `;` means that if you have a cypher query containing `;` then that will also split. 
Test case: `{ str: 'MATCH (n: {foo: "bar;"}) RETURN n', expect: ['MATCH (n: {foo: "bar;"}) RETURN n'] }`